openfisca_core/entities/entity.py

```diff
@@ -1,13 +1,9 @@
-from openfisca_core.types import TaxBenefitSystem, Variable
-from typing import Any, Optional
-
-import os
 import textwrap
 
-from .role import Role
+from ._core_entity import _CoreEntity
 
 
-class Entity:
+class Entity(_CoreEntity):
```

Comment:
It might be relevant to differentiate between `(Single|Singular|Atomic|FindABetterWord)Entity` and `GroupEntity` to be perfectly clear.

Reply:
Yes, absolutely agree. However, that needs to be perfectly well though of and be anounced in advance, as it can probably break every single repository based on core.